src/PrestaShopBundle/Install/Database.php

```diff
@@ -104,6 +104,15 @@ public function testDatabaseSettings($server, $database, $login, $password, $pre
             }
         }
 
+        if (!$errors) {
+            if (($select_error = Db::checkSelectPrivilege($server, $login, $password, $database, $prefix)) !== true) {
+                $errors[] = $this->translator->trans('Your database login does not have the privileges to select data in table from the database "%s". Please ask your hosting provider to have them.', ['%database%' => $database], 'Install');
```

Comment:
Hi there, I've searched about this privilege because I did not know what it was and it looks like the right terminology about this action is _grant CREATE/DROP/etc. privilege_ ([read](https://mediatemple.net/community/products/dv/204404494/how-do-i-grant-privileges-in-mysql)), therefore I suggest the following wordings: 

- _You must be granted the privilege to select data in the tables of database "%s". Ask your hosting provider to enable it._ which should be easier for merchants to understand the issue

- _Your current login must be granted the SELECT privilege for database "%s". Ask your hosting provider to enable it._ which should fit the technical terminology

@Junebyun, what do you think?

Reply:
Hi @LouiseBonnard, thank you for your suggestions! I will go with the first option. :)